docs/source/getting_started/installation/python_env_setup.inc.md

```diff
@@ -14,6 +14,6 @@ Or you can create a new Python environment using [uv](https://docs.astral.sh/uv/
 
 ```console
 # (Recommended) Create a new uv environment. Use `--seed` to install `pip` and `setuptools` in the environment.
-uv venv vllm --python 3.12 --seed
-source vllm/bin/activate
+uv venv --python 3.12 --seed
+source .venv/bin/activate
```

Comment:
random idea: Can we just rename it to `my_vllm` or something like that?

Reply:
Since the venvs created by `uv`are local to each project, I always find myself using the default name. Especially because this is in most `.gitignore` files I can create it inside the repo with no issues.